package.json

```diff
@@ -26,27 +26,25 @@
     "lib"
   ],
   "engines": {
-    "node": ">= 10.0.0"
+    "node": ">= 22.0.0"
```

Comment:
Is it actually `22.0.0`, I think we should set this to the sync-ESM version specifically.

Reply:
Ahh, my bad got confused over 20 vs 22 😅